### ISSUE TITLE
HTML parse from data object

### ISSUE THREAD

**Issue by ivangrimaldo** [kbrsh/moon#242]
Hi Im getting the data prom an api but the content is html formatted, my question is how I parse to render correctly because I'm getting the string not the html inside my view:

```
{
content:" &lt;p&gt;Hi,&lt;/p&gt;↵&lt;p&gt;DigitalOnUs Inc. top 20 fastest IT Services co. as per Inc.500 2018 ratings.&lt;/p&gt;↵&lt;p&gt;As part of our ever growing requirements, "
}
```

**Comment by kbrsh** [comment]
In Moon v1, there is no current way of directly setting `innerHTML`. I am planning support for this with an `innerHTML` attribute.

```html
<div innerHTML={content}></div>
```

**Comment by dingjibang** [comment]
@kbrsh excuse me, Is there a way to set innerHTML for a dom now?

**Comment by kbrsh** [comment]
This was implemented in c206837d6b3b1bf620b801e0c025e1171e5aafc1.
